src/or/dirserv.c

```diff
@@ -1042,6 +1041,37 @@ format_versions_list(config_line_t *ln)
     smartlist_split_string(versions, ln->value, ",",
                            SPLIT_SKIP_SPACE|SPLIT_IGNORE_BLANK, 0);
   }
+
+  /* Handle the case where a dirauth operator has accidentally made some
+   * versions comma-separated instead of space-separated. */
```

Comment:
This comment seems backward. The code looks like it checks for and warns about spaces.